packages/neos-ui-editors/src/Editors/Reference/index.js

```diff
@@ -48,6 +56,7 @@ export default class ReferenceEditor extends PureComponent {
             options={sanitizeOptions(options)}
             value={value}
             onValueChange={this.handleValueChange}
+            onReferenceClick={() => value && setActiveContentCanvasSrc(options[0].uri)}
```

Comment:
As far as I can tell, `onReferenceClick` is never triggered anywhere inside the `<SelectBox/>` implementation.

Looks to me like a leftover from an earlier stage of this PR. So, the prop is probably supposed to be called `onItemClick`, but the `<SelectBox/>` is missing an implementation for that as well.

Reply:
I think we don't need to implement it in <SelectBox/>, instead, we name it onClick.

it works but https://github.com/neos/neos-ui/issues/3057https://github.com/neos/neos-ui/issues/3057 is still there.